schemas/3.0.0-without-$id.json

```
@@ -2542,6 +2542,10 @@
                     "type": "string",
                     "description": "A brief summary of the message."
                 },
+                "version": {
+                    "type": "string",
+                    "description": "A version for the message."
```

Slight suggestion, feel free to ignore:

```suggestion
                    "description": "Version of the message."
```

---

applying this to the definition, as I have wrongly done it in the schema. Learning